implementations/elixir/ockam/ockam_services/test/services/abac_policies_api_test.exs

```diff
@@ -0,0 +1,64 @@
+defmodule Ockam.Services.API.ABAC.PoliciesApi.Test do
+  @moduledoc false
+  use ExUnit.Case
+
+  alias Ockam.ABAC.ActionId
+  alias Ockam.ABAC.PolicyCheck
+  alias Ockam.ABAC.PolicyStorage
+  alias Ockam.ABAC.Request
+
+  alias Ockam.API.Client, as: ApiClient
+  alias Ockam.Services.API.ABAC.PoliciesApi
+
+  setup do
+    Application.put_env(:ockam_abac, :policy_storage, Ockam.ABAC.PolicyStorage.ETS)
```

Comment:
If we're changing the application config in the test, then I think we should restore it to the previous state in the `on_exit/1` callback, otherwise it might affect other tests.